### PR TITLE
DropLog: log pod-names instead of addresses

### PR DESCRIPTION
**Pull request by shastrinator**
If available, use pod-names instead of addresses
in pod events

Signed-off-by: Kiran Shastri <shastrinator@gmail.com>

### CODE REVIEW

File: pkg/hostagent/event_poster.go [hunk 1]
```diff
@@ -173,8 +160,24 @@ func (agent *HostAgent) processPacketEvent(packetEvent PacketEvent, currTime tim
 	if !srcOk && !dstOk {
 		return nil
 	}
-
-	// update poster's eventSubmitTimeMap
-	agent.poster.eventSubmitTimeMap[packetEvent.SourceIP+packetEvent.DestinationIP] = currTime
+	if eventDestination != nil {
+		var eventSrcName, eventDstName string = packetEvent.SourceIP, packetEvent.DestinationIP
```
Comment (jayaramsatya) [inline pkg/hostagent/event_poster.go:164]: Nit: Above lines 160-162, you are checking for srcOk and dskOk  if they are not present you are returning from there,    then you can  directly set the eventSrc and dst names right ?

File: pkg/hostagent/event_poster.go [hunk 2]
```diff
@@ -109,7 +109,7 @@ func (agent *HostAgent) processPacketEvent(packetEvent PacketEvent, currTime tim
 	if agent.shouldIgnore(packetEvent, currTime) {
 		return nil
 	}
-	var srcEventPosted bool = false
+	var eventDestination *v1.Pod = nil
```
Comment (jayaramsatya) [inline pkg/hostagent/event_poster.go:112]: nit: you can assume it is nil unless you set the pointer

File: pkg/hostagent/event_poster.go (hunk 1 above)
Comment (shastrinator) [reply pkg/hostagent/event_poster.go:164]: The pods may be present(srcOk,dstOk), but not in running state, we don't post events in that case.

File: pkg/hostagent/event_poster.go (hunk 2 above)
Comment (shastrinator) [reply pkg/hostagent/event_poster.go:112]: ok, will remove the init.

**Review by jayaramsatya — APPROVED** [review]
LGTM

**Comment by shastrinator** [comment]
I merged this a little earlier than the checks passed, sorry about that. I noticed this later.

**Pull request merged**